go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -98,6 +103,16 @@ func configureUDNServicesNFTables() error {
 	})
 	tx.Flush(&knftables.Chain{Name: nftablesUDNServiceMarkChain})
 
+	tx.Add(&knftables.Chain{
+		Name:    nftablesUDNBGPOutputChain,
+		Comment: knftables.PtrTo("Drop traffic generated locally towards advertised UDN subnets"),
+
+		Type:     knftables.PtrTo(knftables.FilterType),
+		Hook:     knftables.PtrTo(knftables.OutputHook),
+		Priority: knftables.PtrTo(knftables.FilterPriority),
+	})
+	tx.Flush(&knftables.Chain{Name: nftablesUDNBGPOutputChain})
```

Comment:
don't laugh at me :D its my firs time reviewing nft code, what does flush do? why would we add and then immediately flush the chain ? makes no sense
please help me :D

Reply:
So after my recent changes I flush the set on startup and expect it to be populated when the networks are added on startup. 
Although it ensures that we do not have any staleness, there is a slight potential for it to cause a blip where the rules are not there. I think this is acceptable for now.